src/configuration/OpenSearchDetails.tsx

```diff
@@ -89,7 +82,9 @@ export const OpenSearchDetails = (props: Props) => {
           <VerticalGroup>
             <div>
               The plugin uses the configured version below to construct the queries it sends to the connected OpenSearch
-              instance. If the configured version does not match the instance version, there could be query errors.
+              instance. If the configured version does not match the instance version, there could be query errors. If
+              your OpenSearch instance is configured with &quot;Compatibility Mode&quot;, it will auto-set as
+              &quot;Elasticsearch 7.10.2&quot;. You can correct it with the dropdown menu.
```

Comment:
I wonder if this should be a bit more detailed - "you can correct it" doesn't tell me whether or when I _should_ correct it. We don't expect any issues in this case, right? And since the first part of this is about possible problems with version conflicts it might be clearer to have this be on a separate line.